tools/docker/docker-installation-fedora-22-29.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Getting started with Docker on Fedora
+subsection: docker
+order: 1
```

Comment:
You cannot have both pages ordered first.

What is the case for current Docker? is this page relevant?

Reply:
I believe this page is relevant for anyone using a legacy version of Fedora. However, if this page is best moved to another location that provides other legacy information please let me know and I will update the pull request. 

for the time being - would you prefer that I remove this page from the pull request?